brainscore/metrics/spatial_correlation.py

```diff
@@ -0,0 +1,54 @@
+import numpy as np
+from brainscore.metrics import Score, Metric
+
+
+def _score(scores):
+    center = np.mean(scores)
+    error = np.std(scores)
+    score = Score([center, error], coords={'aggregation': ['center', 'error']}, dims=('aggregation',))
+    score.attrs[Score.RAW_VALUES_KEY] = scores
+
+    return score
+
+
+class SpatialCorrelationSimilarity(Metric):
+
+    def __init__(self, metric, bin_size_mm):
+        '''
+        Computes the similarity of two given distributions using a given metric. The metric is applied to each bin which
+        in turn are created based on a given bin size and the independent variable of the distributions
+
+        :param metric: metric to be applied to each bin
+        :param bin_size_mm: size per bin in mm | one fixed size, utilize Score.RAW_VALUES_KEY to change weighting
+        '''
+        self._metric = metric

```

Comment:
maybe this should be called `similarity_function` to not overload the metric term